base.lua

```diff
@@ -334,7 +331,7 @@ function courseplay:onDraw()
 	local isDriving = self:getIsCourseplayDriving();
 
 	--WORKWIDTH DISPLAY
-	if self.cp.mode ~= 7 and self.cp.timers.showWorkWidth and self.cp.timers.showWorkWidth > 0 then
+	if self.cp.settings.driverMode:get() ~= 7 and self.cp.timers.showWorkWidth and self.cp.timers.showWorkWidth > 0 then
```

Comment:
@pvaiko Is it correct, that the work width isn't shown for the bale collector ai driver ?

Reply:
it is, it does not make any sense there. While you are at it, could replace the magic number with the constant, btw